Portal/test/Datahub.Functions.UnitTests/EmailServiceTests.cs

```diff
@@ -1,8 +1,8 @@
-﻿using Moq;
-using System.Collections.Generic;
-using Datahub.Functions.Services;
+﻿using Datahub.Functions.Services;
 using Microsoft.Extensions.Logging;
-using Datahub.Infrastructure.Queues.Messages;
+using Moq;

```

Comment:
?

Reply:
Can't say I know where this comes from or why to be honest, will revert